certbot/plugins/enhancement_test.py

```diff
@@ -0,0 +1,179 @@
+"""Tests for new style enhancements"""
```

Comment:
We still need to do https://github.com/certbot/certbot/pull/5912#discussion_r194743614.